### ISSUE TITLE
Print loss on the terminal through the slim.learning.train () function

### ISSUE THREAD

**Issue by VideoImage** [tensorflow/magenta#962]
How to print the loss value on the terminal, such as ：INFO:tensorflow:global step 2511: loss = 2510.0000 (1.71 sec/step).

**Comment by superxc** [comment]
the same question
